README.md

```diff
@@ -1,8 +1,24 @@
 [![Build Status](https://travis-ci.org/CODAIT/MAX-Object-Detector-Web-App.svg?branch=master)](https://travis-ci.org/CODAIT/MAX-Object-Detector-Web-App)
 
-# Object Detector Web App
+# Create a web app to visually interact with objects detected with machine learning
+
```

Comment:
I guess this will be updated later perhaps, but I think the opening paragraphs should be more descriptive (ref: https://github.com/IBM/MAX-Image-Caption-Generator-Web-App/#create-a-web-app-to-interact-with-machine-learning-generated-image-captions)

Reply:
yeah the one line will be replaced with multiple paragraphs today, that's the one WIP task, I just wanted eyes on the rest of the changes while I'm working on it today.